.golangci.yml

```diff
@@ -12,4 +12,6 @@ linters:
 run:
   deadline: 1m
 
-
+linters-settings:
+  errcheck:
+    ignore: fmt:.*,gopkg.in/go-playground/validator.v9:.*RegisterValidation
```

Comment:
```
$ /usr/local/bin/golangci-lint run 
ERRO Can't read config: can't unmarshal config by viper: 1 error(s) decoding:

* 'linters-settings.Errcheck.ignore' expected a map, got 'string' 
```

golangci-lint `v1.12.3` is error. but `v1.12.5` is success.
Probably CI's Worker version is not `v1.12.5`.